.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,20 +1,48 @@
----
-name: Feature request
-about: Suggest an idea for this project
-title: ''
-labels: ''
-assignees: ''
+name: 🛠️ Feature request
```

Comment:
This appears to still be a `.md` file, I think you need a `git mv`

Reply:
Yeah ! apologies for the oversight. I've corrected it now.